### PR TITLE
Remove superfluous relationship.

### PR DESCRIPTION
**Pull request by jordandukart**

### CODE REVIEW

File: api/islandora_xacml_api.install [hunk 1]
```diff
@@ -110,3 +110,86 @@ function islandora_xacml_api_update_7102(&$sandbox) {
...
+
+  $result = array_pop($sandbox['result_stash']);
+  $datastream_uri = $result['ds']['value'];
+
+  $parts = explode('/', $datastream_uri);
+  $pid = $parts[0];
+  $ds = $parts[1];
+  $object_to_update = islandora_object_load($pid);
+
+  // Need to either replace the existing RELS-INT if a POLICY still exists
```
Comment (willtp87) [inline api/islandora_xacml_api.install]: Comment out of date.

File: api/islandora_xacml_api.install [hunk 2]
```diff
@@ -110,3 +110,83 @@ function islandora_xacml_api_update_7102(&$sandbox) {
...
+
+  $parts = explode('/', $datastream_uri);
+  $pid = $parts[0];
+  $ds = $parts[1];
+  $object_to_update = islandora_object_load($pid);
+
+  $manageable_by_user = 'isManageableByUser';
+  $manageable_by_role = 'isManageableByRole';
+  $object_to_update[$ds]->relationships->remove(ISLANDORA_RELS_EXT_URI, $manageable_by_user);
+  $object_to_update[$ds]->relationships->remove(ISLANDORA_RELS_EXT_URI, $manageable_by_role);
```
Comment (whikloj) [inline api/islandora_xacml_api.install:179]: Starting to update to 1.5 and just want to confirm this is removing the superfluous RELS-INT relationship but using the RELS-EXT prefix?

Comment (willtp87) [inline api/islandora_xacml_api.install:179]: Work is still moving on this https://github.com/Islandora/islandora_xacml_editor/pull/115

Comment (whikloj) [inline api/islandora_xacml_api.install:179]: Sorry, does that mean that I shouldn't use this drush script because the underlying changes are not complete?

Comment (willtp87) [inline api/islandora_xacml_api.install:179]: This is not a Drush script. No, it's a little more complex than that and this update has been committed and the changes Nelson is working on will need to handle the case where this has been run.

Comment (whikloj) [inline api/islandora_xacml_api.install:179]: Apologies for my confusion.

**Pull request merged**